app.py

```diff
@@ -330,9 +331,18 @@ def clipCollection(image):
     collection = collection.map(clipCollection)
     return collection
 
+# File Parser: GPKG (.gpkg)
+def parse_gpkg_coords_to_ee_geometry(file):
+    # load GPKG file data to fiona collection
+    with fiona.open(fp=file) as f:
```

Comment:
fiona fails to open the temporary file for some reason which ends up in a DriverError. i tried to work it out by passing a tmp_path instead of tmp.name, adding `delete=False` and flushing it resolved this issue soemhow so it moves to the next line which also crashes